meson.build

```diff
@@ -363,13 +363,24 @@ if get_option('enable-app')
   endif
 endif
 
-if get_option('enable-nnstreamer-tensor-filter')
-  if get_option('platform') == 'android'
-    warning('android nnstreamer-filter is not yet supported, building nnstreamer-filter skipped')
-  else
-    nnstreamer_dep = dependency('nnstreamer', required: true)
-    subdir('nnstreamer/tensor_filter')
-  endif
+if get_option('enable-nnstreamer-tensor-filter').enabled()
+	if get_option('platform') == 'android'
+		warning('android nnstreamer-filter is not yet supported, building nnstreamer-filter skipped')
+	else
+    nnstreamer_filter_dep = dependency('nnstreamer')
+		subdir('nnstreamer/tensor_filter')
+		extra_defines += '-DSUPPORT_NNSREAMER_PLUGIN_FILTER=1'
```

Comment:
Which parts require this define statement?

Reply:
I am currently not using it though.
I added it to remove dependency when header is not install such as ```ML_API_COMMON```
I will check the need again and delete it if it is not needed.
Thank you.
```
#if ML_API_COMMON
#include <ml-api-common.h>
#else
/**
 @ref:
 https://gitlab.freedesktop.org/dude/gst-plugins-base/commit/89095e7f91cfbfe625ec2522da49053f1f98baf8
 */
#if !defined(ESTRPIPE)
#define ESTRPIPE EPIPE
#endif /* !defined(ESTRPIPE) */

#define _ERROR_UNKNOWN (-1073741824LL)
#define TIZEN_ERROR_TIMED_OUT (TIZEN_ERROR_UNKNOWN + 1)
#define TIZEN_ERROR_NOT_SUPPORTED (TIZEN_ERROR_UNKNOWN + 2)
#define TIZEN_ERROR_PERMISSION_DENIED (-EACCES)
#define TIZEN_ERROR_OUT_OF_MEMORY (-ENOMEM)
typedef enum {
  ML_ERROR_NONE = 0,                    /**< Success! */
  ML_ERROR_INVALID_PARAMETER = -EINVAL, /**< Invalid parameter */
  ML_ERROR_TRY_AGAIN =
    -EAGAIN, /**< The pipeline is not ready, yet (not negotiated, yet) */
  ML_ERROR_UNKNOWN = _ERROR_UNKNOWN,         /**< Unknown error */
  ML_ERROR_TIMED_OUT = (_ERROR_UNKNOWN + 1), /**< Time out */
  ML_ERROR_NOT_SUPPORTED =
    (_ERROR_UNKNOWN + 2),               /**< The feature is not supported */
  ML_ERROR_PERMISSION_DENIED = -EACCES, /**< Permission denied */
  ML_ERROR_OUT_OF_MEMORY = -ENOMEM,     /**< Out of memory (Since 6.0) */
} ml_error_e;
#endif
```